core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -339,6 +352,16 @@ swagger_params(PathMeta) ->
                               ])
     end.
 
+auth_token_param(<<"/accounts/{ACCOUNT_ID}/",_/binary>>, _Method) ->
```

Comment:
I think the clicktocall endpoint doesn't require an auth token when executing the clicktocall; its toggle-able whether the auth token is required. So maybe add a clause for the /connect of a clicktocall and have it be optional.